web/src/features/panels/zone/MethodologyCard.tsx

```diff
@@ -0,0 +1,45 @@
+import Accordion from 'components/Accordion';
+import { RoundedCard } from 'features/charts/bar-breakdown/GraphCard';
+import { t } from 'i18next';
+import { EmapsIcon } from 'icons/emapsIcon';
+import trackEvent from 'utils/analytics';
+
+export default function MethodologyCard() {
+  return (
+    <RoundedCard>
+      <Accordion
+        icon={<EmapsIcon />}
+        title={t('left-panel.applied-methodologies.title')}
+        className="pb-1 pt-3 text-md"
+        onOpen={() => trackEvent('AppliedMethodologies Expanded')}
+      >
+        <div className="flex flex-col gap-2 pb-1 text-emerald-800 underline underline-offset-4 dark:text-emerald-500">
+          <a
+            href="https://www.electricitymaps.com/methodology#missing-data"
+            target="_blank"
+            rel="noreferrer"
+            className={`text-sm font-semibold `}
+          >
+            <span>{t('left-panel.applied-methodologies.estimations')}</span>
+          </a>
+          <a
+            href="https://www.electricitymaps.com/methodology#data-collection-and-processing"
+            target="_blank"
+            rel="noreferrer"
+            className={`text-sm font-semibold `}
+          >
+            <span>{t('left-panel.applied-methodologies.flowtracing')}</span>
+          </a>
+          <a
+            href="https://www.electricitymaps.com/methodology#carbon-intensity-and-emission-factors"
+            target="_blank"
+            rel="noreferrer"
+            className={`text-sm font-semibold `}
+          >
```

Comment:
Should we add some form of hover effect here to indicate these are external links like we do on the buttons in the navbar?
<img width="128" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/30777521/0e8e7437-eb85-46ac-90e2-313b4cfef6d2">

Reply:
Personally I don't think so because it is a link and not a button. What do you think @Alportan ?

Reply:
Agreed with Silke, since they are simple links, it should be fine. We might thing of adding an icon in the future if needed to indicate that it opens an external website, but I don't think it's a blocker for this PR 🙌